### PR TITLE
Updated roadmap

### PR DESCRIPTION
**Pull request by edoardo123-PAY**

### CODE REVIEW

**Review by nlsteers — APPROVED** [review]

File: source/roadmap.html.erb [hunk 1]
```diff
@@ -11,57 +11,82 @@ title: Roadmap
...
         </p>
-        <h3 class="govuk-heading-m">Infrastructure migration</h3>
-        <p class="govuk-body">We’ve migrated our infrastructure to AWS Fargate, making sure we can scale and maintain the platform in an efficient and resilient way.</p>
-        <h3 class="govuk-heading-m">Improved payment pages</h3>
+        <h3 class="govuk-heading-m">Improve how we handle data</h3>
         <p class="govuk-body">
-          We’ve revised the journey for creating
-          <%= link_to 'payment pages', 'govuk-payment-pages', class: 'govuk-link' %>, and making a payment using them. Users are now asked for one thing per page, and services can
-          <%= link_to 'provide links with the amount and reference pre-filled', 'https://docs.payments.service.gov.uk/prefill_payment_links', class: 'govuk-link' %>.
+          We're planning to get the best new card payment suppliers, to make sure service teams and paying users have the best value for money and experience when using GOV.UK Pay.
```
Comment (Beth-Brown) [inline source/roadmap.html.erb:78]: This line duplicates line 74

**Review by Beth-Brown — CHANGES_REQUESTED** [review]
There's one line that looks like it's been repeated (line 74 and 78)

File: source/roadmap.html.erb (hunk 1 above)
Comment (edoardo123-PAY) [reply source/roadmap.html.erb:78]: Thanks Beth ! Great spot!

File: source/roadmap.html.erb [hunk 2]
```diff
@@ -11,57 +11,92 @@ title: Roadmap
...
+        <h2 class="govuk-heading-l">Future plans</h2>
+        <h3 class="govuk-heading-m">Procuring new card payment suppliers</h3>
         <p class="govuk-body">
-          Users will be able to agree to allow a service to take payments in the future without their intervention, for example to pay for a licence that renews on a monthly or
-          annual basis.
+         We're planning to get the best new card payment suppliers, to make sure service teams and paying users have the best value for money and experience when using GOV.UK Pay.
+        </p>
+        <h3 class="govuk-heading-m">Improve how we handle data</h3>
+        <p class="govuk-body">
+         We're planning to get the best new card payment suppliers, to make sure service teams and paying users have the best value for money and experience when using GOV.UK Pay.
```
Comment (edoardo123-PAY) [inline source/roadmap.html.erb]: Need to change this

Comment (edoardo123-PAY) [reply source/roadmap.html.erb]: to this "We want to define a data organisation model to make Pay more efficient, improving performance and providing better value for money."

**Comment by edoardo123-PAY** [comment]
Had to create a new branch with the changes, as there was an issue with the current branch.  Please look at this one instead - https://github.com/alphagov/pay-product-page/pull/694

**Pull request closed without merging**